src/zimscraperlib/zim/creator.py

```diff
@@ -146,7 +150,31 @@ def config_indexing(
         self.__indexing_configured = True
         return self
 
+    def _maybe_metadata_from_maybe_illustration(self, name, value):
+        if not isinstance(name, str) or not name.startswith("Illustration_"):
+            return None
+        if isinstance(value, bytes):
+            image = io.BytesIO(value)
+        try:
+            img = PIL.Image.open(image)
+            return img.format
+        except Exception:
+            return None
+
+    def _log_metadata(self):
+        if logger.isEnabledFor(DEBUG):
+            for name, value in sorted(self._metadata.items()):
+                illustration_metadata = self._maybe_metadata_from_maybe_illustration(
+                    name, value
+                )
+                if illustration_metadata is not None:
+                    logger.debug(f"Metadata: {name} MD = {illustration_metadata}")
```

Comment:
`MD` means nothing to me. I don't get why we add something specific here while the metadata for illustration is supposed to already be pre-computed.

I would simplify the whole block by getting rid of the if/else block: `logger.debug(f"Metadata: {name} = {illustration_metadata if illustration_metadata else value}")`